### ISSUE TITLE
Result data plotting in IDE fails due to "missing" module omnetpp.scave.results

### ISSUE THREAD

**Issue by thor** [omnetpp/omnetpp#796]
**Platform**
 - OMNeT++ version: 6.0pre6
 - OS: Arch Linux

**Describe the bug**
Attempting to generate plots after opening a result file such as `*.vec` results in a failure to load the Python module `omnetpp.scave`. The issue evolves around struggling to find the correct Python modules where it expects them, though I don't know why this happens. <sup>1</sup>

```
# ...
  File "/opt/omnetpp/ide/plugins/org.omnetpp.scave.pychart_5.3.0.200103-547749b72b/python/omnetpp/internal/PythonEntryPoint.py", line 18, in <module>
    from omnetpp.scave import results
ImportError: cannot import name 'results' from 'omnetpp.scave' (unknown location)
```

![image](https://user-images.githubusercontent.com/234489/75110919-82256b80-5634-11ea-9ddd-43fdeae3a7e3.png)

Knowing that the `sys.path` is an easy way to gauge what's up, I see that the output of `sys.path` contains `/opt/omnetpp/ide/plugins/org.omnetpp.scave.pychart_5.3.0.200103-547749b72b/python`, but not `/opt/omnetpp/python`, where the module `omnetpp.scave` also resides. I get a bit further if I append `/opt/omnetpp/python` to `sys.path`, but then it fails to load `omnetpp.scave.impl_ide`.
That happens because by that point it's looking in `/opt/omnetpp/python/omnetpp/scave` for the module `impl_ide`, because that's where it first found `omnetpp.scave`, _I think_.

I realise the plot feature is quite new, so perhaps I'm doing something quite wrong too.

(*Sidenote: it would've been really nice if errors like these are also sent to `stderr`, as it'd allow me to easily copy and paste it from `error.log`, but that's neither here nor there!)

**To Reproduce**
1. Create a result file such as `Tictoc17-#0.vec` with the `tictoc` sample project.
2. Open it with the IDE.
3. Go to the _"Browse Data"_ tab.
4. Either _right-click_ or use the _button in upper right corner_ to _"Plot"_ the data.
5. Observe error message appear.

**Environment**
Per the instructions in `pythoncharts` I created a virtual environment with the required Python packages thusly:
1. Create `pyproject.toml` with the following contents:
   ```toml
   [tool.poetry]
   name = "simulations"
   version = "0.1.0"
   description = ""
   
   [tool.poetry.dependencies]
   python = "^3.8"
   py4j = "0.10.7"
   numpy = "^1.18.1"
   pandas = "^1.0.1"
   matplotlib = "^3.1.3"
   seaborn = "^0.10.0"
   
   [build-system]
   requires = ["poetry>=0.12"]
   build-backend = "poetry.masonry.api"
   ```
2. Install the dependencies into (by default) a virtual environment:
   ```sh
   poetry install
   ```
3. Start OMNeT++ in the virtual environment:
   ```sh
   poetry run omnetpp
   ```

---
1. The line numbers are off because I commented out the `sys.path` print statements I insert myself to see what's happening.

**Comment by torokati44** [comment]
There are indeed two locations that need to be in the `PYTHONPATH` of the `python3` processes for the result analysis to work.

The first one is of course the `python` folder in the root of the OMNeT++ installation. This is set up in the shell (used to launch the IDE) by the sourcing of the `setenv` script. Perhaps entering the virtual environment overwrites this. You could try sourcing `setenv` within the virtualenv, or adding this directory to `PYTHONPATH` manually. This is then passed through the IDE into the spawned `python3` processes normally.
I'm not sure that appending it to `sys.path` when the process already started to execute the script is "soon enough", and maybe the order of these entries matters.

I don't think that previous imports (where the already imported modules were found) influences the lookup order/procedure of modules imported later.

The other necessary location is, as you also found out, the `ide/plugins/org.omnetpp.scave.pychart_<version>/python/` directory. This is added to the environment of the `python3` processes dynamically by the IDE. (Because it has to be a full (absolute) path, which also changes depending on whether the IDE is in a "deployed" state, as it is when started from a release tarball, or in a "development" state, started from another Eclipse instance.)
I'm not sure how entering a virtualenv can affect this, since this location is "discovered" based on the actual location of the corresponding Eclipse plugin (Java bundle, or what they call it...?).

Additionally, the directory of the `.anf` file is also added automatically by the IDE, to make importing little reusable snippets easier, but omitting this should not cause problems unless you wish to import local `.py` files in your chart scripts.

Thank you for testing the preview release, for the detailed description of the issue, and for the additional tips about what to do with the error messages.

**Comment by torokati44** [comment]
Fixed for 6.0.
